client/app/dedicated/server/bandwidth/dedicated-server-bandwitdh.routing.js

```diff
@@ -0,0 +1,82 @@
+export default /* @ngInject */ ($stateProvider) => {
```

Comment:
This routing file is a bit huge. May be confusing.

Reply:
Some states are basically the same, i've simplified it